drop-transfer/src/ws/client/mod.rs

```diff
@@ -269,37 +260,14 @@ async fn make_request(
     Err(err)
 }
 
-async fn tcp_connect<Fut: Future<Output = bool>>(
-    local_ip: IpAddr,
-    remote_ip: IpAddr,
-    break_check: impl Fn() -> Fut,
-    logger: &Logger,
-) -> Option<TcpStream> {
+async fn tcp_connect(local_ip: IpAddr, remote_ip: IpAddr, logger: &Logger) -> Option<TcpStream> {
```

Comment:
Is this function worth keeping? It's so insignificant at this point that it would be less cumbersome to just inline it